multi_agent_path_planning/lifelong_MAPF/lifelong_MAPF.py

```diff
@@ -7,6 +8,26 @@
 from multi_agent_path_planning.lifelong_MAPF.mapf_solver import BaseMAPFSolver
 from multi_agent_path_planning.lifelong_MAPF.task_allocator import BaseTaskAllocator
 from multi_agent_path_planning.lifelong_MAPF.task_factory import BaseTaskFactory
+from multi_agent_path_planning.lifelong_MAPF.helpers import *
```

Comment:
nitpick: not my favorite bc in confuses IDEs and can lead to collisions in large projects.